bulk_reassign_docs.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python3
+
+import argparse
+import requests
+from utils import SigmaClient
+
+
+def get_member_id(client, user_email):
+    """ Update member
+
+        :access_token:  Generated access token
+        :user_email:    Users email
+
+        :returns:       ID associated with the user_email
+    """
+    try:
+        response = client.get(
+            f"v2/members?search={user_email}"
+        )
+        response.raise_for_status()
+    # HTTP and other errors are handled generally by raising them as exceptions to be surfaced in downstream code
+    except requests.exceptions.HTTPError as errh:
+        # The API response's message value is sent in lieu of the full response to display to the user for clarity
+        raise Exception(errh.response.status_code,
+                        f"API message: {errh.response.json()['message']}")
+    except requests.exceptions.ConnectionError as errc:
+        raise Exception(
+            f"Connection Error: {errc}, API response: {errc.response.text}")
+    except requests.exceptions.Timeout as errt:
+        raise Exception(
+            f"Timeout Error: {errt}, API response: {errt.response.text}")
+    except requests.exceptions.RequestException as err:
+        raise Exception(
+            f"Other Error: {err}, API response: {err.response.text}")
+    else:
+        data = response.json()
+        if len(data['entries']) == 0:
+            print("No users found with this email:", user_email)
+            raise SystemExit("Script aborted")
+        else:
+            return data['entries'][0]['memberId']
+
+
+def get_member_files(client, user_id):
+    """ Update member
+        :user_id:       id of current owner
+
+        :returns:       an array of file objects
+
+    """
+    data = []
+    moreResults = True
+    nextPage = ''
+    while moreResults:
+        try:
+            response = client.get(
+                f"v2/members/{user_id}/files?typeFilters=workbook&typeFilters=dataset&limit=500{nextPage}"
```

Comment:
hmmm i read the docs as this being required 🤔 
<img width="670" alt="image" src="https://github.com/user-attachments/assets/e1df6106-a8a7-4eb1-abb7-fcbbf8e3be18">

Reply:
I actually think this might be outdated on the docs side, because if you omit the `limit` param entirely, you'll get 50 results and the `nextPage` token too

Reply:
@BrettBSigma Never mind, turns out I'm wrong - you can ignore this one 😂

Reply:
Actually... strangely, it looks like you can set the limit to even higher than 500 and it still works, though... maybe there's something wrong with the endpoint here